### PR TITLE
ROX-24480: Config Mgmt e2e test fixes

### PR DESCRIPTION
**Pull request by alwayshooin**
testing

### CODE REVIEW

File: ui/apps/platform/cypress/integration/configmanagement/ConfigurationManagement.helpers.js [hunk 1]
```diff
@@ -255,9 +255,13 @@ export function interactAndWaitForConfigurationManagementScan(interactionCallbac
 }
 
 export function navigateToSingleEntityPage(entitiesKey) {
-    interactAndWaitForConfigurationManagementEntityPage(() => {
-        cy.get('[data-testid="side-panel"] [aria-label="External link"]').click();
-    }, entitiesKey);
+    // interactAndWaitForConfigurationManagementEntityPage(() => {
```
Comment (alwayshooin) [inline ui/apps/platform/cypress/integration/configmanagement/ConfigurationManagement.helpers.js:258]: upon investigation with @pedrottimark -- testing hypothesis that the gql request is caching on slow connections and thus not making the second request once the URL lands on the detail page

**Comment by pedrottimark** [comment]
ocp-4-12-ui-e2e-tests failure is known and fixed by #11574

> Workload CVE Image CVE Single page should correctly handle local filters on the deployments tab: should correctly handle local filters on the deployments tab

Timed out retrying after 4050ms: `cy.click()` failed because this element:

`<button aria-disabled="false" class="pf-v5-c-button pf-m-link" type="button" data-ouia-component-type="PF5/Button" data-ouia-safe="true" data-ouia-component-id="OUIA-Generated-Button-link-1">Clear f...</button>`

is being covered by another element:

`<span class="pf-v5-c-menu__item-text">No resu...</span>`

**Comment by pedrottimark** [comment]
Rerun ui-e2e-tests:

/test gke-ui-e2e-tests
/test ocp-4-12-ui-e2e-tests
/test ocp-4-15-ui-e2e-tests

File: ui/apps/platform/cypress/integration/configmanagement/ConfigurationManagement.helpers.js (hunk 1 above)
Comment (pedrottimark) [reply ui/apps/platform/cypress/integration/configmanagement/ConfigurationManagement.helpers.js:258]: Reran the tests.

This change seems worthwhile:
* with comment here
* optional comment out `function interactAndWaitForConfigurationManagementEntityPage` with reason why

**Review by pedrottimark — APPROVED** [review]

**Pull request merged**